concepts/05 UI Components/HtmlEditor/20 Keyboard Support.md

```diff
@@ -0,0 +1,90 @@
+An end user can use the following keys to interact with the UI component.
+
+<table class="dx-table">
+    <tr>
+        <th>PC</th>
+        <th>Mac</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>Ctrl + C</td>
+        <td>&#8984; + C</td>
+        <td>Copy</td>
+    </tr>
+    <tr>
+        <td>Ctrl + V</td>
+        <td>&#8984; + V</td>
+        <td>Paste</td>
+    </tr>
+    <tr>
+        <td>Ctrl + Z</td>
+        <td>&#8984; + Z</td>
+        <td>Undo</td>
+    </tr>
+    <tr>
+        <td>Ctrl + Y</td>
+        <td>&#8984; + Y</td>
+        <td>Redo</td>
+    </tr>
+    <tr>
+        <td>Ctrl + A</td>
+        <td>&#8984; + A</td>
+        <td>Select All</td>
+    </tr>
+    <tr>
+        <td>Ctrl + B</td>
+        <td>&#8984; + B</td>
+        <td>Bold</td>
+    </tr>
+    <tr>
+        <td>Ctrl + I</td>
+        <td>&#8984; + I</td>
+        <td>Italic</td>
+    </tr>
+    <tr>
+        <td colspan="2">Enter</td>
+        <td>Insert a line break (e.g. a new paragraph)</td>
```

Comment:
Firstly, `e.g.` needs a comma after, secondly, you should not use Latin abbreviations.